spec/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "user"
+require 'db_helpers'
+RSpec.describe User do
+  describe "New user" do
+    it "creates a new user" do
+      newuser = instance_double(User)
+      allow(newuser).to receive(:username).and_return("test")
+      allow(newuser).to receive(:email).and_return("test@test.com")
+      allow(newuser).to receive(:password).and_return("test")
+      expect(newuser.username).to eq "test"
+      expect(newuser.email).to eq "test@test.com"
+      expect(newuser.password).to eq "test"
+    end
+  end
+
```

Comment:
First test appears to be a false positive - it's creating a mock (in lines 6 to 9) and the testing that mock (in lines 10 to 12). It doesn't actually use the user.rb code

Reply:
this test is now redundant so we have removed it